std/experimental/allocator/package.d

```diff
@@ -1099,7 +1099,11 @@ void dispose(A, T)(auto ref A alloc, T p)
 if (is(T == class) || is(T == interface))
 {
     if (!p) return;
-    auto support = (cast(void*) p)[0 .. typeid(p).initializer.length];
+    static if (is(T==interface))
+        auto ob = cast(Object) p;
```

Comment:
@MEtaLang.

Actually I think it's not a problem. COM interfaces are reference counted, so trying to free one with `dispose` is a programming error. So there is no problem.